client/src/components/login_signup/ProfilePage.js

```diff
@@ -1,34 +1,87 @@
 import React, { Component } from "react";
-import { Grid, Cell } from "react-mdl";
+
+import { Table, Button, Form } from "react-bootstrap";
+import { FormRow } from "react-bootstrap/Form";
+import { Grid, Row, Col } from "react-bootstrap";
 
 export default class profilePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      file: null
+    };
+    this.uploadSingleFile = this.uploadSingleFile.bind(this);
+    this.upload = this.upload.bind(this);
+  }
+
+  uploadSingleFile(e) {
+    this.setState({
+      file: URL.createObjectURL(e.target.files[0])
+    });
+  }
+
+  upload(e) {
+    e.preventDefault();
+    console.log(this.state.file);
+  }
+
   render() {
+    let imgPreview;
+
+    if (this.state.file) {
+      imgPreview = (
+        <img src={this.state.file} alt="avatar" className="avatar-img" />
+      );
+    }
     return (
       <div style={{ width: "80%", margin: "auto" }}>
-        <Grid className="landing-grid middle">
-          <Cell col={1}>
-            <img
-              src="https://www.intrahealth.org/sites/ihweb/files/styles/large/public/person-thumbnail-images/avatar-20_0.png?itok=mM4-Q4b-"
-              alt="avatar"
-              className="avatar-img"
-            />
-          </Cell>
-          <Cell col={2}>
-            <div className="form-group">
-              <label>First name</label>
-              <input type="text" className="form-control" />
-              <label>Last name</label>
-              <input type="text" className="form-control" />
-              <label>Email address</label>
-              <input type="email" className="form-control" />
-            </div>
-          </Cell>
-          <p className="text-left">
-            <a href="/changePassword">
-              Change password
-            </a>
-          </p>
-        </Grid>
+        <form>
+          <div className="form-group preview">{imgPreview}</div>
+
+          <div className="form-group">
+            <button>
+              <input type="file" onChange={this.uploadSingleFile} />
+            </button>
+          </div>
+        </form>
+
+        <div className="form-group">
+          <label>First name</label>
+          <input type="text" className="form-control" placeholder="Vardenis" />
+          <label>Last name</label>
+          <input
+            type="text"
+            className="form-control"
+            placeholder="Pavardenis"
+          />
+          <label>Email address</label>
+          <input
+            type="email"
+            className="form-control"
+            placeholder="s@gmail.com"
+          />
+          <Form>
+            <Form.Group>
+              <Form.Label>City</Form.Label>
+              <Form.Control as="select">
```

Comment:
should use an array list which could be saved in a file or somewhere upfront in the code